GoogleDataTransportCCTSupport/GDTCCTTests/Unit/GDTCCTPrioritizerTest.m

```diff
@@ -168,9 +168,11 @@ - (void)testNetworkConnectionInfo {
   GDTCCTPrioritizer *prioritizer = [[GDTCCTPrioritizer alloc] init];
   GDTCOREvent *event = [_CCTGenerator generateEvent:GDTCOREventQosDefault];
   event.needsNetworkConnectionInfoPopulated = YES;
+  event.eventCode = [[NSNumber alloc] initWithInt:1405];
   [prioritizer prioritizeEvent:event];
   NSData *networkConnectionInfoData = event.networkConnectionInfoData;
   XCTAssertNotNil(networkConnectionInfoData);
+  XCTAssertNotNil(event.eventCode);
   gdt_cct_NetworkConnectionInfo info;
   [networkConnectionInfoData getBytes:&info length:networkConnectionInfoData.length];
   XCTAssertNotEqual(info.network_type, gdt_cct_NetworkConnectionInfo_NetworkType_NONE);
```

Comment:
I think you should also test eventCode's value after niling out networkConnectionInfo, as I found above, eventCode info could be lost when the needsNetworkConnectionInfo property is set to nil.